### PR TITLE
Add ECS for 8.9 FF

### PR DESCRIPTION
**Pull request by bhapas**

### CODE REVIEW

File: shared/versions/stack/master.asciidoc [hunk 1]
```diff
@@ -13,7 +13,7 @@ bare_version never includes -alpha or -beta
 :prev-major-version:     7.x
 :prev-major-last:        7.17
 :major-version-only:     8
-:ecs_version:            master
+:ecs_version:            8.9
```
Comment (kgeller) [inline shared/versions/stack/master.asciidoc:16]: Can someone from docs verify this is ok? 

We noticed that as a part of the [8.8 FF PR](https://github.com/elastic/docs/pull/2672) this was changed to master from the specific version. Previously, our team (ECS) has always set this ([8.7 FF PR](https://github.com/elastic/docs/pull/2669/files))

Comment (kilfoyle) [reply shared/versions/stack/master.asciidoc:16]: Yeah, that looks like it indeed should be `8.9`. Nice catch!

**Review by kilfoyle — APPROVED** [review]
LGTM! 🎸

File: shared/versions/stack/master.asciidoc (hunk 1 above)
Comment (kgeller) [reply shared/versions/stack/master.asciidoc:16]: Awesome, thanks @kilfoyle !

Comment (kilfoyle) [reply shared/versions/stack/master.asciidoc:16]: Let me check on this...

Comment (kilfoyle) [reply shared/versions/stack/master.asciidoc:16]: Checked again. Yes indeed, it should be `8.9`. :-)

**Review by kilfoyle — APPROVED** [review]
LGTM! ⛵

**Pull request merged**